tree/ntuple/v7/doc/tuning.md

```diff
@@ -5,15 +5,15 @@ A cluster contains all the data of a given event range.
 As clusters are usually compressed and tied to event boundaries, an exact size cannot be enforced.
 Instead, RNTuple uses a *target size* for the compressed data as a guideline for when to flush a cluster.
 
-The default cluster target size is 100 MB of compressed data.
+The default cluster target size is 128 MiB of compressed data.
 The default can be changed by the `RNTupleWriteOptions`.
 The default should work well in the majority of cases.
 In general, larger clusters provide room for more and larger pages and should improve compression ratio and speed.
 However, clusters also need to be buffered during write and (partially) during read,
 so larger clusters increase the memory footprint.
 
 A second option in `RNTupleWriteOptions` specifies the maximum uncompressed cluster size.
-The default is 1 GiB.
+The default is 10x the default cluster target size, i.e. ~1.2 GiB.
```

Comment:
Why not
```suggestion
The default is 8x the default cluster target size, i.e. 1.0 GiB.
```
?
Or why the factor `10`?

Reply:
Was the reason for choosing `10` something else than 'get to a round number'?  If it is just to get to a round number we should switch to 8, if it is something else we should explain it here.

Reply:
Well, the largest compression factor we have seen in an experiment EDM so far was MiniAOD with a compression factor >9.